### DISCUSSION TITLE
Rebuilding working grafana server after *.js changes

### DISCUSSION THREAD

**Discussion by elektryx109** [grafana/mimir#6080] [Help and support] [answered]
Hello, I am looking for help or someone who can show me how to rebuild working grafana server (on ubuntu)
I want to change some js file (disable data export etc), I know what I need to modify but I can't find way to rebuild this changes to server.
I lost two days on searching solutions and now I give up, I need someone smarter in js apps...
Best

**Comment by charleskorn** [comment, marked as answer]
Is this related to Mimir specifically? It sounds like it might be a question about Grafana, in which case you might have more luck asking in github.com/grafana/grafana.

**Reply by elektryx109** [reply]
True, misclick with topic
sorry
